GeneticAlgorithm.java

```diff
@@ -0,0 +1,186 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Random;
+
+
+public class GeneticAlgorithm {
+    private static ArrayList<Genome> population;
+    private int populationSize = 3;
+    private static int numberOfFeatures = 6; //highestCol and lowestCol are replaced by heightDifference
```

Comment:
I think we should keep them separate to retain more information for our model.

Reply:
Just to clarify, do you mean `heightDifferenceWeight` should be kept as `highestCol` and `lowestCol`?